src/client/core/utils/key-char.js

```diff
@@ -239,8 +191,67 @@ export function parseKeysString (keysString) {
     });
 
     return {
-        commands: commands,
-        error:    error,
-        keys:     keysString
+        combinations: combinations,
+        error:        error,
+        keys:         keysString
     };
 }
+
+export function getArrayByKeyCombination (keysCombination) {
```

Comment:
`getArrayByKeyCombination` but `keysCombination`. I suppose it's `keyCombination`, not `keysCombination`